playfab-docs/features/data/dataconnections/index.md

```diff
@@ -27,6 +27,10 @@ You can get started with data connections for event ingestion using PlayFab’s
 
 Data connections is the replacement for the Event Export and Export to S3 features that currently PlayFab supports. Data connections allow for faster export in preferred formats for offline processing. You get to bring your storage account for data to be ingested and distributed. That means you get to manage and control the encryption-at-rest policy, the lifetime management policy, and network access. The billing becomes simplified as the costs associated with that storage account are accounted for centrally in your all-up Azure account billing.  
 
+### Blob Data Connection
 The data is readily available in Parquet blob format. Parquet is a column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk, resulting in low latency, higher throughput, and low cost of data storage. It's designed to be a common interchange format for both batch and interactive workloads. Thus, data connections enables your endeavors for custom advanced analytics and reporting, to run ad-hoc queries in a cost-optimized manner.
 
+### Azure Data Explorer Data Connection
+Data Connections allows you to export to Azure Data Explorer for near-real time ingestion and distribution of your data. Azure Data Explorer provides lightning-fast data exploration capabilities, robust analytics, and the ability to handle large volumes of diverse data in near real-time, enabling you to gain invaluable insights and make informed decisions swiftly and efficiently. 
```

Comment:
@dhavalbhattmsft note that you're contributing to the public repo.

Would be great to quantify "large volumes of diverse data" and "lightning-fast". It might mean different things to different people.

Reply:
@joannaleecy  thank you for your input - I've quantified those details.